charts/metrics-server/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
+name: metrics-server
+description: Metrics Server is a scalable, efficient source of container resource metrics for Kubernetes built-in autoscaling pipelines.
+type: application
+version: 3.3.0
+appVersion: 0.5.0
+keywords:
+  - kubernetes
+  - metrics-server
+  - metrics
+home: https://github.com/kubernetes-sigs/metrics-server
+icon: https://avatars.githubusercontent.com/u/36015203?s=400&v=4
+sources:
+  - https://github.com/kubernetes-sigs/metrics-server
+maintainers:
+  - name: stevehipwell
```

Comment:
Please update with a second maintainer.

Reply:
@krmichelos are you happy to be the second maintainer?

Reply:
If he is not, I'm happy to volunteer. 😃

Reply:
I am fine with it. Please use krmichel as the user name. I am largely afk for about a week.

Reply:
Adding @endrec would give us a third to help out since he said he is willing

Reply:
@krmichel my thoughts exactly. Do either of you want an email address or URL added? I'm adding my GitHub profile as my URL.

Reply:
You can use my gh profile link too as a URL.

Reply:
My gh profile is fine as well

Reply:
Added.

Reply:
If still time, I'm also willing to help maintain.

Reply:
If @serathius is happy for me to add another maintainer and to re-run the tests I'll add you. Otherwise you can always open a PR after this has been merged.

Reply:
@stevehipwell for me it makes sense to close this as soon as possible and then to add/remove maintainers can always be done later.